src/main/java/net/apnic/whowas/App.java

```diff
@@ -112,15 +98,19 @@ public TaskExecutor taskScheduler() {
     ApplicationContext context;
 
     @Autowired
-    private JdbcOperations jdbcOperations;
-
     private RipeDbLoader dbLoader;
+
     private Exception lastDbException = null;
 
     @PostConstruct
-    public void initialise() {
-        dbLoader = new RipeDbLoader(jdbcOperations, -1L);
-        executorService.execute(this::buildTree);
+    public void initialise() throws ExecutionException, InterruptedException {
```

Comment:
This will need to be changed a bit as It looks like still using the old way of loading data before the move of this code?